From 298723358bc9fa8f979969c02e6c54f7552f4f86 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 10:36:52 +0800
Subject: [PATCH 1/1] Test case for driver .nasm file

Description: Change the variable reference value in the MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm b/MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm
index 3a8707616f1a..69c0a5cadb5a 100644
--- a/MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm
+++ b/MdeModulePkg/Universal/EbcDxe/X64/EbcLowLevel.nasm
@@ -56,7 +56,7 @@ ASM_PFX(EbcLLCALLEXNative):
       ;   This aids in the simplicity of supporting C unprototyped functions,
       ;   and vararg C/C++ functions."
       ;
-      cmp    r8, 0x20
+      cmp    r8, 0x10
       jae    skip_expansion
       mov    r8, dword 0x20
 skip_expansion:
-- 
2.27.0.windows.1

